### ISSUE TITLE
Fadecandy does not reconnect if the first try failed on startup

### ISSUE THREAD

**Issue by wongkaiweng** [iron-ox/fadecandy_ros#18]
On the startup of the node, there are times when fadecandy failed to connect and you see a warning:
```
Failed to connect to Fadecandy device: Found Fadecandy device but could not connect; will retry every second
```

However, the fadecandy never connects. The only solution is to restart the node. This is different from #17 that it reconnects when you unplug and replug the fadecandy.

**Comment by reinzor** [comment]
I will look into this

**Comment by reinzor** [comment]
@wongkaiweng , tested this here but I cannot reproduce your issue. 

```
ROS Noetic, 20.04
```

![fc](https://user-images.githubusercontent.com/4176023/108201886-a66d3880-7120-11eb-9fb2-28b34dd5b31a.gif)

**Comment by wongkaiweng** [comment]
Thanks for trying it out @reinzor. On our end, we are on Melodic 18.04. Let me see if I can get a single strip to run an isolated test.

**Comment by reinzor** [comment]
Resolved by #20
